scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,48 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    """
+    Khatri-rao product
+
+    Parameters
+    ----------
+    a:  (I, K) ndarray
```

Comment:
lowercase `(n, k) arraylike` would be the convention for SciPy. If an arraylike is passed instead of a numpy array it should still work. You can see many examples how to handle the conversion.